easybuild/easyconfigs/f/flex/flex-2.6.0-GCC-4.9.2.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
```

Comment:
@valtandor: this should be removed, there's a flex easyblock, cfr. https://github.com/valtandor/easybuild-easyconfigs/pull/8